contrib/gitoxide/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "gitoxide"
+pkgver = "0.34.0"
+pkgrel = 0
+build_style = "cargo"
+make_build_args = [
+    "--no-default-features",
+    "--features=max-control,gix-features/zlib-stock,gitoxide-core-blocking-client,http-client-curl",
+]
+make_install_args = list(make_build_args)
+make_check_args = list(make_install_args)
+hostmakedepends = ["cargo", "cmake", "pkgconf"]
```

Comment:
```suggestion
hostmakedepends = ["cargo", "pkgconf"]
```

not used anymore with the other stuff fixed